pyseir/inference/whitelist_generator.py

```diff
@@ -42,7 +43,7 @@ def __init__(
         self.nonzero_case_datapoints = nonzero_case_datapoints
         self.nonzero_death_datapoints = nonzero_death_datapoints
 
-    def generate_whitelist(self):
+    def generate_whitelist(self, timeseries: TimeseriesDataset) -> pd.DataFrame:
```

Comment:
How long does the new version take?

Reply:
18 seconds in generate_whitelist :-/ Of that 10s is in get_columns_and_date_subset called by calculate_new_case_data_per_region

Reply:
hmm, could using `parallel_apply` from pandarallel speed things up?

Reply:
That reduced the wall time from 20s to 13s.